Include/internal/pycore_flowgraph.h

```diff
@@ -41,6 +41,9 @@ PyAPI_FUNC(PyObject*) _PyCompile_OptimizeCfg(
         PyObject *consts,
         int nlocals);
 
+// Export for '_testinternalcapi' shared extension
+PyAPI_FUNC(PyObject*) _PyCompile_OptimizeLoadFast(PyObject *instructions);
```

Comment:
We don't tend to expose individual peephole optimisations, we test them through OptimizeCFG (see DirectCfgOptimizerTests). Is there a reason why you can't do that for this optimization?

Reply:
Sure, I can add a call to `optimize_load_fast` into `_PyCompile_OptimizeCFG`. I wasn't sure if `_PyCompile_OptimizeCFG` was intended to only test `_PyCfg_OptimizeCodeUnit`. We want this pass to be called as late as possible in compilation, which is why it lives outside of `_PyCfg_OptimizeCodeUnit`.